tests/test_helpers.py

```diff
@@ -5,7 +5,7 @@
 
 import pytest
 
-from capellambse import helpers
+from capellambse import MelodyModel, helpers
```

Comment:
The `MelodyModel` import is no longer used in this test module.

```suggestion
from capellambse import helpers
```